pom.xml

```diff
@@ -63,12 +63,12 @@
         <insights-notification-schemas-java.version>0.20</insights-notification-schemas-java.version>
         <clowder-quarkus-config-source.version>1.5.0</clowder-quarkus-config-source.version>
```

Comment:
This needs to be bumped to a Quarkus 3 version before the merge.

Reply:
Release done in clowder-quarkus-config-source. Now, we need to wait until it's available in Maven Central.